portfolio/src/main/java/com/google/sps/servlets/DeleteTaskServlet.java

```diff
@@ -0,0 +1,26 @@
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for deleting tasks. */
+@WebServlet("/delete-task")
+public class DeleteTaskServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Actually, this should be a DELETE request. Is it part of the tutorial that ask you to do a post here?

Reply:
Hi! In the tutorial, there was an example that showed us to delete comments with post. https://github.com/clairezc27/step/blob/master/walkthroughs/week-3-server/examples/todo-list/src/main/java/com/google/sps/servlets/DeleteTaskServlet.java

Reply:
Got it! Just keep in mind DELETE http request is actually more appropriate to be used here :D